src/helpers/format-chat-history.ts

```diff
@@ -376,6 +377,28 @@ async function processNodeContent(
 
           tryAddContent(codeLines, testTokenLimits);
         }
+
+        const regex = /https:\/\/docs\.google\.com\/(spreadsheets|document|presentation)\/d\/[a-zA-Z0-9_-]+/g;
+
+        const matches = comment.body.match(regex);
+        const hasGoogleLink = !!matches?.length;
+
+        // Pull google content
+        if (hasGoogleLink) {
+          await Promise.all(
+            matches.map(async (match) => {
+              const resp = await getContentFromUrl(match);
+              if (resp) {
+                const contentXml = `
+                  <google-doc href="${matches[0]}" title="${resp.title}">
```

Comment:
I think maybe this should be based on the type of document

Doc
Sheet
Slides

Or if it doesn't matter then maybe standardize using 

`<google-drive>`